src/main/java/org/tndata/android/compass/activity/MyPrioritiesActivity.java

```diff
@@ -62,10 +82,38 @@ private void swapFragments(int index, boolean addToStack, Category category){
             if (addToStack){
                 mFragmentStack.add(fragment);
             }
-            getFragmentManager().beginTransaction().replace(R.id.base_content, fragment).commit();
+            if (index == CATEGORIES){
+                mToolbar.setTitle("My Priorities");
```

Comment:
lets put this in `strings.xml` somewhere. There may already be a value for this (e.g. the title of the nav drawer menu).